simulation/agents.py

```diff
@@ -50,6 +54,7 @@ def __init__(
         property_type: PropertyType,
         built_form: BuiltForm,
         heating_system: HeatingSystem,
+        heating_system_install_date: datetime.datetime,
```

Comment:
Why is `household.heating_system_install_date` a `datetime` and not a `date`?

Reply:
True, I'll change it to a date, no need to have a lifetime in microseconds!